health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/Boundary.js

```
@@ -0,0 +1,210 @@
+        )
+    }
+    else
+    {
+        return (
+            <React.Fragment>
+                {showPopUp &&  (
+                    <PopUp 
+                        className={"custom-popup-boundary"}
+                        type={"default"}
+                        heading={t("CHOOSE_MEANS_TO_CREATE_BOUNDARY")}
+                        children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)